modules/ROOT/partials/integrations/vue-quick-start.adoc

```diff
@@ -167,7 +167,7 @@ import Editor from '@tinymce/tinymce-vue'
     <img alt="Vue logo" class="logo" src="./assets/logo.svg" width="125" height="125" />
     <editor
       id="uuid"
-      licenseKey="gpl"
+      licenseKey="gpl" // gpl for open source, T8LK:... for commercial
```

Comment:
Comments are allowed in JSX apparently, but I believe you need a different syntax:
```suggestion
      licenseKey="gpl" /* gpl for open source, T8LK:... for commercial */
```